plugins/autocomplete/plugin.js

```diff
@@ -744,9 +751,17 @@
 				viewHeight = this.element.getSize( 'height' ),
 				editable = editor.editable(),
 				// Bounding rect where the view should fit (visible editor viewport).
-				editorViewportRect = editable.isInline() ? editable.getClientRect( true ) : editor.window.getFrame().getClientRect( true ),
-				// How much space is there for the view above and below the specified rect.
-				spaceAbove = rect.top - editorViewportRect.top,
+				editorViewportRect;
+
+			// iOS classic editor has different viewport element (#1910).
+			if ( CKEDITOR.env.iOS && !editable.isInline() ) {
+				editorViewportRect = CKEDITOR.document.getById( editor.id + '_contents' ).getClientRect( true );
```

Comment:
It looks to me that the original code (`editorViewportRect = editor.window.getFrame().getClientRect( true );`) should work correctly.

If it doesn't isn't it actually a Safari bug? It should be reported then in upstream tracker and referenced.